systemtest/instrumentation_test.go

```diff
@@ -89,9 +88,9 @@ func TestAPMServerInstrumentation(t *testing.T) {
 		if !ok {
 			continue
 		}
-		assert.Equal(t, transactionDoc.Trace.ID, traceID)
-		assert.Equal(t, transactionDoc.Transaction.ID, entry.Fields["transaction.id"])
-		return
+		if transactionDoc.Trace.ID == traceID && transactionDoc.Transaction.ID == entry.Fields["transaction.id"] {
+			return
+		}
```

Comment:
apm-server is emitting 1 more log in the trace before the expected log due to elasticsearch cache not ready on startup.